### ISSUE TITLE
Github Pages

### ISSUE THREAD

**Issue by Gismo1337** [Gismo1337/vue-cookie-consent-banner#25]
## **Is your feature request related to a problem? Please describe.**

* Request

---

## **Describe the solution you'd like**

* Please add a github pages template in a new folder `/docs` to get a project website

---

## **Describe alternatives you've considered**

*

---

### **Additional context**

*

**Comment by nabeel001** [comment]
hey @Gismo1337, can you pls elaborate what you are looking for exactly in a github page template.

**Comment by Gismo1337** [comment]
feel free to bring your own ideas into this issue.

I think it's important to explain the cookie component and link the npm package included a small tutorial how to install and use.

nothing special

**Comment by shashankbhatgs** [comment]
@Gismo1337 Is it picking already existing GitHub page templates from https://pages.github.com/themes and then adding it in a docs folder?

**Comment by Gismo1337** [comment]
or create a own. both works @shashankbhatgs

**Comment by shashankbhatgs** [comment]
ok please assign it to me @Gismo1337 
Also if I find any obstacles can I ask for help here itself?

**Comment by Gismo1337** [comment]
i assigned you both @nabeel001  @shashankbhatgs  please get in touch witch each other. dont waste your time :D

**Comment by shashankbhatgs** [comment]
@Gismo1337 sorry im finding it hard to add a template page. Couldn't figure out how to add yml file. @nabeel001 you can work on it. I've unassigned myself.

**Comment by DarkFalc0n** [comment]
Can I work on this issue?

**Comment by Gismo1337** [comment]
> Can I work on this issue? @Gismo1337 
> 

go for it @DarkFalc0n

**Comment by HridoyHazard** [comment]
what's the update ?? @DarkFalc0n  ?? if you are busy can i work on it ??

**Comment by DarkFalc0n** [comment]
> what's the update ?? @DarkFalc0n ?? if you are busy can i work on it ??

Yes, Go on @HridoyHazard. I am busy currently and will be unassigning myself.
